### DISCUSSION TITLE
offline keep usage

### DISCUSSION THREAD

**Discussion by momoadc** [keephq/keep#636] [Q&A] [answered]
Is keep supposed to work on an on-prem env? 
I am having a bit of trouble (providers section not loading, logs indicate that it is searching online...)
There is no documentation on the subject so I would love some help :)

**Comment by shahargl** [comment, marked as answer]
Yes! It does. We have some users deployed Keep in air gapped environments. Could you join our slack where we can help you out? https://slack.keephq.dev
